### ISSUE TITLE
Port forwarding not working on Windows 10 (Hyper-V)

### ISSUE THREAD

**Issue by gingeleski** [gingeleski/vagrant-ghost#5]
I'd kind of realized the port forwarding wasn't working (of 2368 on guest to 8080 on host) but ignored it thus far.

Now that this project is playing a bigger role in how I maintain my own Ghost blog, easy access to the locally running Ghost instance is crucial.

This issue covers a workaround to the following known problem:

- https://github.com/hashicorp/vagrant/issues/9750
- https://www.vagrantup.com/docs/hyperv/limitations.html

Basically, I don't care if the port doesn't end up mapping, but I need to be able to hit `http://<some_host_resolvable_address>:2368` to get at "the Ghost in the box." 🙂
